apis/install/webhook/binoculars_webhook.go

```diff
@@ -14,31 +14,44 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package v1alpha1
+package webhook
```

Comment:
it seems like running `make generate` will now recreate this file in the v1alpha1

Reply:
I don't think `make generate` does the webhook code at all.  That was manually added.

Reply:
Why is this being moved out of the `v1alpha1` folder though? Isn't the webhook versioned along with the rest of the api code?

It may not be modified when running `make generate` but the file was still created using the `operator-sdk create` command. Or am I mistaken?

Reply:
I'm not trying to prescribe one way or the other. I just wonder why we're deviating from what the `operator-sdk` prescribed.

Reply:
I'm going to revert this and just aim to test the webhooks via integration.  

I moved it out so I could put any generated code (that we won't test) in v1alpha and the webhook code would be tested in a separate directory.  @dejanzele pointed out a problem with this esp with auto generation and new generation via operator-sdk.  

The stubs are created by the operator-sdk but I think you do have to manually add your own logic in there so I'm not sure how valuable the operator-sdk generation is once we have a few templates to follow. 

TBH I still don't see the value in the webhooks.  They seem to be a lot harder to setup and we only use them to fill in some minor defaults.